File: wrapUpTasks/java/PrintAllArmstrongNumbersFrom1To1000.java
public class PrintAllArmstrongNumbersFrom1To1000{

    public static void main(String[] args){
        int number = 1000;
    
        for(int count = 1; count < number; count++){
            if(isArmstrongNumber(count)){
                System.out.print(count+" ");
            }
            
        }
        System.out.println();
    } 
    
    public static boolean isArmstrongNumber(int number){
    
        int original = number;
        int digitCount = 0;
        int sum = 0;
        int temp = original;
        
        while(temp > 0){
        
            temp /=  10;
            digitCount++;
        }
        
        while(number > 0){
        
            int remainder = number % 10;
            sum += Math.pow(remainder, digitCount);
            number /= 10;
        }
        return sum == original;
            
    }
}

File: wrapUpTasks/javaScript/printAllArmstrongNumbersFrom1To1000.js

function isArmstrongNumber(number) {

	let original = number;
	let digitCount = 0;
	let sum = 0;
	let temp = original;

	while(temp > 0){
		temp = Math.floor(temp/10);
		digitCount++;
	}

	while(number > 0){

		let remainder = number % 10;
		sum += Math.pow(remainder, digitCount);
		number = Math.floor(number/10);
	}
	return sum == original;
}

let number = 1000;
for(let count = 1; count <= number; count++){

	if(isArmstrongNumber(count)){
		console.log(count+" ");
	}
}
console.log();